lib/rpm_package.py

```diff
@@ -151,7 +151,7 @@ def _load(self):
 
             # load distro files
             files = self.package_data.get('files', {}).get(
-                distro_attrib_name, {}).get(self.distro.version, {})
+                distro_attrib_name, {}).get(self.distro.version, {}) or {}
```

Comment:
Got it. I agree in merging this as a precaution, but I think it's better to delete the empty values still.